CHANGELOG.md

```diff
@@ -70,7 +70,7 @@ New Features
 - `SED`s can now be constructed with flexible units via the `astropy.units`
   module. (#789).
 - Added new surface brightness profile, 'InclinedExponential'. This represents
-  the 2D projection of the 3D profile:
+  the 2D projection the 3D profile:
```

Comment:
Per my comment to rmjarvis, this was due to me mistakenly committing a fix of this to the master instead of this branch.